src/app/public/modules/phone-field/phone-field.component.scss

```diff
@@ -0,0 +1,41 @@
NEdaTVCsG2BGDvNSbxVHeHvR3Xy0JIAqzdsJWYpzMk5gJ8dAc9v9fNps0RoBlYaDCJC7NJxg23ZPd872NXjUqE26eE7uVv4IFTEJ+ECPgkXAFCkRTnC28SxGQ38rW0C7Q4H7YV6n8JtKPVBQoVQ2OCnrRdyceESDmj288uWGcQjCYSm7Bdys0d+8ubw4jYn77/ztr/sSa83h8LcEK3t2WT55lAyfiw+Xx7RdHR8dHeU1N4VkeRukeyUvaqIoekDR52T/MISjIPPZezsyWx98iFy6ztQnRaqKVEVafej34spE0KgaBaJjiijppXx63NncMrSj2m+cyFBKdDzc0i2tpE3fiKJH1/MrZOaeHTVAxA3UL1ZNpHwHh96xZEp7E6REC5l/YdxLnRvZ/N6yYoVsF7kcHl4KpysD9BVfrfx1MAvG02VJCrXQspvTlSuxWiq2HtCYQlbW6ppDLWS7fajKSpT88cQNxI4NZ2valS0DkzuYhouCrNOQkpBS8RJeyLOVzlGLJ/eksSWrKV5/VZanBksFAWhRueELzL0ZLON+koAEROqtBDHIq0kpBjaugK8isTtVCjIsQmL3E7V3q97aacppsAwuyntuy9jABgShGYTkQ1R0A9Fx5AyOy1DcNxoe5++V5lE0eSAqd1CQrvL4jdHdFLnM/eufiVVY3X8bQv7Fd8KkDQtcvxuXn/8dxR5HCRNaZPk7d61cneFP6Zkf6OD3b9t5SCaKlcTHO9AVXPwVmWQlAfs1rxu8M3i9/uJx+MsXWqTLGZnZVFcWMgLmzezxojxO18xs30umjerbL7Ch39eEmepibPUQnHL3jn5lgVaHkrWcSQb6qn/yf20/PVRrFgcZ64fKe2G4ghlWOfEMAzGjhrNxJKxrNu6FocvE9M0aQsHkal82r58i7Z7nxgZW1ZVSRoGXq+XYCBAZmYmrS1byAhXk2vWMrFccmpOFRtbirm/ZRKJmEnAcvDX2slsieRwVv7HvJ84hpBe0GOnDinuLiwaq6cxJXsDjR2LiUSyoONwXlh7ANBTLyoqRIOSLzaa3HGJoL4NFq9J7lLa+2DrKBJq/SqPz/Xw1pQYiRVPw/JhNJ4kaELycYVg3M4IiDiGYeGKRgiPEbBx7z8rmPTRHMvv87WE5aA+UpwObzffvmvJoHaOadhErIoa3CdyLiEEqq4hlP+ywMe+AdG7fqoqViwKWX5co4pRTQt1dCnG59vtjS4UENbQrjvM4+nx1+y7Gthti8zu594NafLqmo94jY8xpImUEIiFURQFr8OFaVkkLRO35iAYD9ux0b4upUgkmRCXnDn7S7531Et49E389TMfL+8tkTBgBVUuOKWdi+bUU3FLBcFXmnDl51Bw3FFkH30czmOOhVTtahTY2lrLzpZqtrbWsKm1mq0tNexsr6c22EI4Guo9R51JWNpK7ZYKfvX06n3OlX+7alYvfEFTTGZmV1HoDtAY9bO2vRzDUtMk17UJijP+8Mtd9kWXjbEve+/1Z1NbINVIXvbWF4ZhIVIx7O6PVSXNIZ2r71yDQNLSbiEcvf3/O7l7WM/DiyNtk6dyj4SUxOMGQV826osLWRV3YupO5r7xEoEzLiOaMFGQ+1wTv8/zsTXYw0qVUuJwuPhiayWdwQBjxxyGGVEAQUbRLALV7/Da2q1YExLE49HdbJ00J0ra9adTckGk4i5TcgVOFcL7cGQsKVleGWV0roPZZTYx1PKKKMsro/vUyBQByXYFIyhwl9tFs0bQQbI9vb0hAYdm0N5ehJLRTHzsu7RO2sov/BnclO1CQbI3lTWWgKyA4IFL47x3phPX2gk2T4HobcHL3TAX0S+kZNf3baoMDjwdagmZOQfZuZyieND3d99v5G0itdsYa06jRAvTEY7TUPEuvulZtKkTuTnjRN7PysHl1NM7EvV39I0RqVqfv6f7+u46TTETLJp2PkkcXLb6MWSdDpa9Wi5Pkrw8H6NGldj2lCKQCYlSqWO+8i4fzj6Hink3cfEb96LK9P3Yrji2lFDX6eKVjUWcvSDEPG87hmFgZER4+Yti6gKuHk1/hsTRJEHRBTtbEvztV8cggLpADMUp0oKh/pNHxrxAT/9Z2si8qQjmjE/SVhYekl//9gjf7xl/OmMQAZLc7bQutAv7hyK8jARjR5dT4K/iisue4fqvr4Y4EAOnu6dvpiIhAUcdXMVvrl7ErX/+GpY3htCNHvIiEokQC2zhFxdmc/5YH7wGjeuSWGEGL1gaxMwcmy2ZuyJCnRWjsNjifhc2pisUO3fu/3U/GR5X1rRQVA2Z5WfHj+7ihh9eQ/nzz/ONSy7BiMW4/7d3U3lZLX9/+o9Er/0ZwYYmHNmZyGRPDh2HtLBM2+4Tu0r2sSTEzVSdmAqhZAaqMNkRmsi20GQm+TYyxb+ep6uuREqBphg98mme4e89lGgXdc/u+suJwSg6h6Q7u7ZnyBRk6RY/Lmxn+pYm7vhDJ4uW2R/SRSbs0iU/+abC/IuO4O7od3lr+6GgJlD0GOYetZ/PnHfHsK7PzD/ZP6/rT58CpcCfdZ3s7BxW/PvfyLlz+YmiUMPgFZ+O6oEJkJOpf8OBIhqmbeaOyXWQ71PZ0pRg3lg3bWEDl64gBCn8c//f70crm3bpe5GVqh1oQUqJy6Gia105vEMTatfX/2n/o7WmSU4WHH30GWx471dsWFfPUTf9Ead0DPlarYZjKFZ9v8Opa5QU+GhoCVHd2Nl9xtsCUcoKMykp8NERiBE3zL3G/rtz/6RECoUsb5DjiwvJDCwh0fQRJ+YW8i93AJRMpNz1OQ5NpTTbw0c7WvjBgjGE27JZ01DPgcecT058ER1ZWzFDdjxcy2gjp2wyytgzWfbuFlCaOHZK8ZDmJ1mp0fqgHzXXIuNkm7ctusxJ64N+zCZ10Mk2THj8lTi3/I+LZxcl2FhhceuVbuaOlry9Es56NQcOLAWftksgDzjsDqWiKz62Zy7dPmJS1n5O9hmf28aRY6qYM6qBPG8ESwqmF
```

Comment:
Yeesh. This will add a lot to the bundle size. Is there another way to tackle this?

Reply:
Nope :( Bobby and I actually had a meeting around this and decided that this was the best method. I don't have the numbers at the moment but it didn't add as much as we thought that it might to the bundle size. I can get them again if needed.